scripts/shell-completion/bash/tofu

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+_tofu_completion() {
+  local cur prev words cword
+  
+  # Check if bash-completion is available
+  if type _get_comp_words_by_ref >/dev/null 2>&1; then
+    _get_comp_words_by_ref -n : cur prev words cword
+  else
+    # Fallback if _get_comp_words_by_ref isn't available
+    cur="${COMP_WORDS[COMP_CWORD]}"
+    prev="${COMP_WORDS[COMP_CWORD-1]}"
+    words=( "${COMP_WORDS[@]}" )
+    cword=$COMP_CWORD
+  fi
+
+  # Check if we're at the end of a command with empty current word
+  # This prevents repeating commands when pressing TAB at the end of a complete command
+  if [[ -z "$cur" && $cword -ge 2 ]]; then
+    # If the last word is already a complete command, don't offer completions
+    # Let OpenTofu's built-in completion handle subcommands
+    COMPREPLY=()
+    return 0
+  fi
+  
+  # Special case for flags that require values
+  if [[ "$prev" == "-var-file" || "$prev" == "--var-file" ]]; then
+    COMPREPLY=($(compgen -f -X '!*.tfvars' -- "$cur"))
+    return 0
+  elif [[ "$prev" == "-chdir" || "$prev" == "--chdir" ]]; then
+    COMPREPLY=($(compgen -d -- "$cur"))
+    return 0
+  fi
+  
+  # Let OpenTofu's built-in completion handle subcommands
+  # If we're completing a flag, just use common flags as OpenTofu doesn't complete flags
+  if [[ "$cur" == -* ]]; then
+    # Define only common flags to reduce maintenance overhead
+    local common_flags="-help -version -chdir"
+    
+    COMPREPLY=($(compgen -W "$common_flags" -- "$cur"))
+    return 0
+  fi
+
+  # For everything else (commands and subcommands), use OpenTofu's built-in completion
```

Comment:
The accepted form of this proposal was for this to _exclusively_ rely on OpenTofu's built-in completion, without any additional logic here whatsoever.

Please remove everything from these scripts _except_ the call to the `tofu` executable.

If the `tofu` executable is not available or fails to produce any completions then no completion information should be returned at all. All future improvements to the completion behavior will be made in the Go code exclusively; these scripts should be very small and are unlikely to change ever again once merged.

(I'm commenting on the bash completion here, but this feedback also applies to the zsh completion: neither of them should do anything except running `tofu` and returning whatever it returned.)

Thanks!